build/Makefile.lnx

```diff
@@ -0,0 +1,6 @@
+BINARY_SERVER=alm
+BINARY_CLIENT=alm-cli
+FIND=find
+DBHOST=localhost

```

Comment:
Feel free to fix it if you know how. Since I am on Windows I can't really test this.